### ISSUE TITLE
Unable to run Fathom using URL path

### ISSUE THREAD

**Issue by doenietzomoeilijk** [usefathom/fathom#222]
I'd like to be able to run Fathom as, say, a mounted directory under a given domain. This would enable me to keep the amount of certificates I'm using as low as possible (and not leak a ton of hostnames).

I'm using the official Docker container which exposes port 8089. My nginx setup has this snippet:

```
upstream fathom {                                                                                                                                                          
    server 192.168.2.10:8089;                                                                                                                                              
}
server {
    server_name example.com; # but with my actual domain name, of course :)
    # rest of the config, other locations
                                                                                                                                               
    location /fathom {                                                                                                                                                     
        return 302 /fathom/;                                                                                                                                               
    }                                                                                                                                                                      
    location /fathom/ {                                                                                                                                                    
        proxy_pass http://fathom/;                                                                                                                                         
        proxy_set_header Host $host;                                                                                                                                       
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;                                                                                                       
        proxy_set_header X-Forwarded-Proto $scheme;                                                                                                                        
        proxy_set_header X-Real-IP $remote_addr;                                                                                                                           
    }                                                                                                                                                                      
}
```

This sort of works: I can visit `https://example.com/fathom/` and I get the login screen. I could also grab the tracker.js and so on. However, the frontend appears to be dead set on using `/api/` for the AJAX requests, which result in 404's of course.

Is it possible to get those calls to use a relative path, just like the CSS and basic JS in the HTML? Failing that, would it be able to add a configuration for "base path" or somesuch? Or is there a specific and very good reason that's not possible that I'm not aware of?

**Comment by dannyvankooten** [comment]
Hi @doenietzomoeilijk,

Are you running v1.2.1 or the latest version from GitHub master? There's some related discussion in #77 and I believe this may have been fixed in 1b7ecfe34b3534f123966ea743fe59322311ff44, which is not yet released.

**Comment by doenietzomoeilijk** [comment]
I'm running the release version. I could give the revision you mentioned a go and see how that pans out. 

It also seems like #77 is exactly the issue I'm having, I'm not sure why I didn't see that before when searching, but thanks for pointing it out. 

As far as I'm concerned, this issue is redundant, I'll close it.
